keras_cv/models/task.py

```diff
@@ -93,6 +94,8 @@ def from_preset(
             load_weights: Whether to load pre-trained weights into model.
                 Defaults to `None`, which follows whether the preset has
                 pretrained weights available.
+            input_shape : input shape that will be passed to backbone
+                initialization, Defaults to `None`.
```

Comment:
You might want to clarify that `None` will not be passed to the backbone.

Reply:
done!

Reply:
For maximum clarity, I might say

```python
"""
If `None`, the preset value will be used.
"""
```